packages/config/src/common/projectCountdowns.ts

```diff
@@ -17,7 +17,19 @@ export const PROJECT_COUNTDOWNS = {
       }
     },
   },
-} as const
+  stageOneRequirementsChange: {
+    expiresAt: new UnixTime(1750284000), // 2025-06-19T00:00:00Z
+    getContext: (project: Layer2 | Layer3) => {
```

Comment:
Not that I think this PR should rework it but clearly the implementation here shows that something about this model with `getContext` is broken